src/app/mock_snark_work_coordinator/dumped_spec.ml

```diff
@@ -0,0 +1,7 @@
+type t =
+  { prover : Signature_lib.Public_key.Compressed.Stable.V1.t
+  ; spec :
+      Snark_work_lib.Selector.Single.Spec.Stable.Latest.t One_or_two.Stable.V1.t
+  ; fee : Currency.Fee.Stable.V1.t
```

Comment:
Curiously, we're actually not using the fee here to generate sok_digest. It seems we're fine filling in any fees for simplicity.